src/services/api/routes/model.py

```
@@ -0,0 +1,91 @@
+model_router = APIRouter(prefix="/model", tags=["model"])
+
+
+@model_router.post(path="/entity")
+async def add_entity(request: AddEntityRequest):
+    """
+    Add a single entity (node) to the graph.
+    """
+    import uuid as uuid_lib
+    from src.constants.kg import Node
+    
+    node = Node(
```

Ruff (0.14.10)

Local variable `node` is assigned to but never used

Remove assignment to unused variable `node`

(F841)